spec/lib/pdf_filler/nc_d400_pdf_spec.rb

```diff
@@ -29,6 +30,10 @@
                primary_esigned_at: tomorrow_midnight)
       }
 
+      before do
+        intake.direct_file_data.mailing_apartment = mailing_apartment
+      end
```

Comment:
daffy does not have a apartment number, but wanted to add it to make sure it was getting filled in